src/pages/NewsAdministration/components/NewsEditor.tsx

```diff
@@ -129,6 +137,14 @@ const NewsEditor = ({ newsId, goToNews }: NewsEditorProps) => {
         <Grid container direction='column' wrap='nowrap'>
           <TextField formState={formState} label='Tittel' {...register('title', { required: 'Feltet er påkrevd' })} required />
           <TextField formState={formState} label='Header' {...register('header', { required: 'Feltet er påkrevd' })} required />
+          <UserSearch
+            control={control}
+            formState={formState}
+            helperText='Velg forfatter av nyheten. Når forfatteren er satt er det ikke mulig å endre etterpå.'
+            label='Forfatter'
+            name='creator'
+            required
```

Comment:
Den er ikke required. Vi må også fikse at en ikke kan endre forfatter senere, så hjelpeteksten kan vel da egentlig fjernes
```suggestion
            label='Forfatter'
            name='creator'
```